build/csv/csv-gen.sh

```diff
@@ -37,7 +37,14 @@ function generate_csv() {
 
     "${YQ_CMD_MERGE[@]}" "$CSV_FILE_NAME" "$ASSEMBLE_FILE_OCP"
 
+    # We don't need to include these files in csv as ocs-operator creates its own.
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/rook-ceph-operator-config_v1_configmap.yaml"
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/objectbucket.io_objectbucketclaims.yaml" 2>/dev/null || true
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/objectbucket.io_objectbuckets.yaml" 2>/dev/null || true
+
     mv "$CSV_FILE_NAME" "../../build/csv/"
+    mv "../../build/csv/ceph/$PLATFORM/manifests/"* "../../build/csv/ceph/"
```

Comment:
nit: should be the same, but for readability seems better to keep the * inside the quotes
```suggestion
    mv "../../build/csv/ceph/$PLATFORM/manifests/*" "../../build/csv/ceph/"
```

Reply:
?

Reply:
I thought it will be same too but it gave me ```mv: cannot stat '../../build/csv/ceph/amd64/manifests/*': No such file or directory`` keep `*` in quotes seems like acting as a name